Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/KILLIAN.MSG

```diff
@@ -209,15 +209,14 @@
 {1010}{}{Gizmo's|Gizmos}
 {1011}{}{Pitt|SKUM PITT|SKUMPITT|SKUM PIT|SKUMPIT|SKUM|SCUM PITT|SCUM PIT|SCUMPIT|SCUMPITT|SCUM}
 {1012}{}{Children of the Cathedral|Children|Cathedral|CoC}
-{1013}{}{Claw|Deathclaw|Deathclaws|Claws}
+{1013}{}{Claw|Deathclaw|Deathclaw|Claw}

```

Comment:
2nd Claws had 2 on purpose.

Reply:
Me stupid. What was I thinking when I removed the plurals?